src/globalvariables.cpp

```diff
@@ -117,9 +117,10 @@ const float GlobalVariables::JoyAxis::JOYSPEED = 20.0;
 
 const QString GlobalVariables::JoyAxis::xmlName = "axis";
 
-const int GlobalVariables::HapticTriggerPs5::CLICKSTRENGTH = 7;
-const int GlobalVariables::HapticTriggerPs5::RIGIDSTRENGTH = 4;
-const int GlobalVariables::HapticTriggerPs5::VIBRATIONSTRENGTH = 7;
+const int GlobalVariables::HapticTriggerPs5::CLICKSTRENGTH = 256;
+const int GlobalVariables::HapticTriggerPs5::RIGIDSTRENGTH = 128;
+const int GlobalVariables::HapticTriggerPs5::RIGIDGRADIENTSTRENGTH = 256;
+const int GlobalVariables::HapticTriggerPs5::VIBRATIONSTRENGTH = 64;
 const int GlobalVariables::HapticTriggerPs5::RANGE = 320;
```

Comment:
[Fix scaling of PS5 haptic feedback strength](https://github.com/AntiMicroX/antimicrox/pull/540/commits/d06c7cf7c0661a09977149f506e303bdac726783)   

What was wrong with scaling before?